graphics/graphics-text.md

```diff
@@ -0,0 +1,39 @@
+# Displaying text
+
+The Graphic Entity Module includes two different class to display text, here's how they work.
+
+## Text
+
+The basic class to display text, it will be displayed as a label on you screen.
+
+```java
+graphicEntityModule.createText("Hello World")
+    .setFontFamily("Lato")
+    .setStrokeThickness(5) // Adding an outline
+    .setStrokeColor(0xffffff) // a white outline
+    .setFontSize(50)
+    .setFillColor(0x000000); // An setting the text color to black
+```
+You'll see in this example that you have a lot of options to manipulate the text, but this class has one downside :
+the font is not garanteed to be the good one, since the client browser has to provide it.
+So it doesn't support a lot of fonts and your font will probably be replaced by the default one of the browser.
```

Comment:
The only font family values that will work are: `"Lato"` and those that are available by default on all browsers. If the browser does not recognise the font family, it will be displayed in the browser's a fallback font.

Reply:
f88be963064532676446da0096ac87cb27408428